libenv/sysinfo.c

```diff
@@ -1085,14 +1085,24 @@ static void OSClasses(EvalContext *ctx)
 
     if (os_release_json != NULL)
     {
-        EvalContextVariablePutSpecial(ctx, SPECIAL_SCOPE_SYS, "os_release", os_release_json, CF_DATA_TYPE_CONTAINER, "source=agent,os,release,derived_from=/etc/os-release");
+        EvalContextVariablePutSpecial(ctx, SPECIAL_SCOPE_SYS, "os_release",
+                                      os_release_json, CF_DATA_TYPE_CONTAINER,
+                                      "source=agent,os,release,derived_from=/etc/os-release");
```

Comment:
I believe it's `derived-from` with dash, not underscore. Also you'd better use `derived-from-file`. Pinging @tzz to verify, he is the tag master. ;-)